### PR TITLE
refactor(actions): Migrate to use new datahub-actions container

### PR DESCRIPTION
**Pull request by jjoyce0510**
In this PR, we migrate the use the newly published [datahub-actions](https://hub.docker.com/r/acryldata/datahub-actions) container (published as part of the public [DataHub Actions](https://github.com/acryldata/datahub-actions) repository).

The configurations (env variables) should be compatible with what exists in current actions image. 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docker/docker-compose-with-cassandra.yml [hunk 1]
```diff
@@ -148,7 +148,7 @@ services:
       - ${HOME}/.datahub/plugins:/etc/datahub/plugins
 
   datahub-actions:
-    image: public.ecr.aws/datahub/acryl-datahub-actions:${ACTIONS_VERSION:-v0.0.1-beta.11}
+    image: public.ecr.aws/datahub/datahub-actions:${ACTIONS_VERSION:-v0.0.1-beta.11}
```
Comment (dexter-mh-lee) [inline docker/docker-compose-with-cassandra.yml:151]: any reason for using ecr one here?

Comment (anshbansal) [reply docker/docker-compose-with-cassandra.yml:151]: And the image tag should be as per the new ones, right?
https://github.com/acryldata/datahub-actions/releases

**Comment by anshbansal** [comment]
> should be compatible

Have you tested this on local?

File: docker/docker-compose-with-cassandra.yml (hunk 1 above)
Comment (jjoyce0510) [reply docker/docker-compose-with-cassandra.yml:151]: good catch this should not be ecr

> > should be compatible
> 
> Have you tested this on local?

Yep - test this locally with same configs (env) we use with quickstart

**Review by anshbansal — APPROVED** [review]
LGTM. Will wait for checks to pass then merge

**Pull request merged**